content/source/assets/stylesheets/terraform-overview/pages/_home.scss

```diff
@@ -368,3 +369,14 @@
     }
   }
 }
+
+#terraform-overview-sign-up-link {
+  & a {
+    background-color: $terraform;
+    color: $white !important;
```

Comment:
I will forgive this only because we are going to re-platform 😅

Reply:
I know, it's horrible... 😢